deploy/group_vars/all.yml

```diff
@@ -86,7 +86,7 @@ k8s_aws_fluent_bit_chart_version: "0.1.18"
 
 k8s_auth_host: "{{ ClusterEndpoint }}"
 k8s_auth_ssl_ca_cert: "k8s_auth_ssl_ca_cert.txt"
-k8s_namespace: "{{ app_name }}-{{ env_name }}"
+k8s_cluster_name: "{{ app_name }}-{{ env_name }}"
```

Comment:
```suggestion
k8s_namespace: "{{ app_name }}-{{ env_name }}"
```

I'm not sure I understand this one. `k8s_namespace` needs to remain. Perhaps `k8s_cluster_name` should also be added here as `philly-hip-stack-cluster`?

<img width="574" alt="image" src="https://user-images.githubusercontent.com/129679/222540421-a2c04b95-66f1-4713-a2ad-6e70a6712987.png">